module5/TaskListApp/TaskListApp/Views/TaskListView.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  TaskListView.swift
+//  TaskListApp
+//
+//  Created by Peter Rice on 12/22/23.
+//
+
+import SwiftUI
+
+struct TaskListView: View {
+  @ObservedObject var tasksStore: TasksStore
```

Comment:
Curious how to manage performance of UI updates triggered by changes in larger applications - such that the entire UI doesnt get re-rendered every time one property in an @ObservedObject changes?

Would a large Store class w/ many @Published properties would trigger a re-render of all views that depend on it as an @ObservedObject regardless of which properties they reference?

Perhaps there is some material on strategies for efficient state management for lager SwiftUI applications/

Reply:
Look at a `@StateObject`. https://www.avanderlee.com/swiftui/stateobject-observedobject-differences/